src/setup_env.sh

```diff
@@ -48,9 +48,12 @@ fi
 
 if [ ! -d $CURRENT_DIR/lib/OpenNMT-py ]; then
   echo "Cloning OpenNMT-py"
-  git clone https://github.com/chenzimin/OpenNMT-py.git $CURRENT_DIR/lib/OpenNMT-py
+  # git clone https://github.com/chenzimin/OpenNMT-py.git $CURRENT_DIR/lib/OpenNMT-py
+  git clone https://github.com/OpenNMT/OpenNMT-py.git $CURRENT_DIR/lib/OpenNMT-py
   echo "Installing requirements for OpenNMT-py"
-  pip3 install -r $CURRENT_DIR/lib/OpenNMT-py/requirements.txt
+  cd $CURRENT_DIR/lib/OpenNMT-py
+  pip install -r requirements.txt
+  python setup.py install --force
```

Comment:
Can you use python3 here

Reply:
I tried to use Python3 here but then it fails in installing libs.

Reply:
As I have to install libs via `pip3`, I use `python3` here